packages/api-gateway/src/drivers/http/routes/admin-panel/index.js

```diff
@@ -1,8 +1,8 @@
 const { adminPanelAdapters } = require('../../../adapters');
```

Comment:
There is an error here, the path is wrong.
```suggestion
const { adminPanelAdapters } = require('../../adapters');
```